scripts/legislation/africa/rwanda_french.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+function remove_all_text_before_first_chapter_header {
+  sed -n '/CHAPITRE PREMIER - DES DISPOSITIONS GENERALES /,$p'
+}
+
+function amend_errors_in_headers {
+  sed -E 's/^(Section [0-9]+ - [A-Za-z,’ ]+)[^a-z]+Article ([0-9]+)(er)?/\1\n\n(\2)/' | \
+    sed -E 's/Article premier:/\n\n(1)/' | \
+    sed -E 's/^(CHAPITRE )I -l:/\1II -/' | \
+    sed -E 's/^(TITRE )I -l:/\1II -/' | \
+    sed -E "s/(CHAPITRE PREMIER - DE L'ENVIRONNEMENT NATUREL )/\1\n\n/" | \
+    sed -E 's/^(Section première: Le sol.*) Article 11:/\1\n\n(11)/' | \
+    sed -E 's/^(Section 3 - De la diversité.*) Article 20:/\1\n\n(20)/' | \
+    sed -E "s/^(Section 4 - De l'atmosphère.*) Article 25:/\1\n\n(25)/" | \
+    sed -E "s/^(Section 2 - Obligations.*) Article 60:/\1\n\n(60)/" | \
+    sed -E 's/^(CHAPITRE )I -ll:/\1III -/' 
+}
+
+function reformat_numbering_within_articles {
+  sed -E 's/([0-9])(°)/[\1]/g' 
+}
+
+function amend_errors_in_articles {   
+  #Article 1
+  amend_error_in_article 1 'bien être' 'bien-être' | \
+  #Article 4
```

Comment:
Please indent by two spaces all lines in this function from here down (inclusive).